recipes/python2-secrets/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
I've opened a PR to include this in the PyPI tarball:
https://github.com/scaleway/python2-secrets/pull/4